config/initializers/rollbar.rb

```diff
@@ -7,6 +7,15 @@
   # Here we'll disable in 'test':
   config.enabled = false if Rails.env.test? || Rails.env.development?
 
+  config.js_enabled = true
+  config.js_options = {
+    accessToken: ENV.fetch('ROLLBAR_CLIENT_ERRORS_ACCESS_TOKEN', ''),
+    captureUncaught: true,
+    payload: {
+      environment: Rails.env
```

Comment:
This should match the behaviour on [line 75](https://github.com/DFE-Digital/teacher-vacancy-service/pull/1064/files#diff-81333f286aa517165a0eceba9340fed1R75).